internal/venice-common/src/main/java/com/linkedin/venice/meta/Store.java

```diff
@@ -248,9 +248,12 @@ static boolean isSystemStore(String storeName) {
 
   void setBootstrapToOnlineTimeoutInHours(int bootstrapToOnlineTimeoutInHours);
 
-  boolean isLeaderFollowerModelEnabled();
+  default boolean isLeaderFollowerModelEnabled() {
+    return true;
+  }
 
-  void setLeaderFollowerModelEnabled(boolean leaderFollowerModelEnabled);
+  default void setLeaderFollowerModelEnabled(boolean leaderFollowerModelEnabled) {
+  }
```

Comment:
Can you add a `@deprecated` annotation to both of these?

Reply:
Done